core/src/main/java/com/segment/analytics/kotlin/core/utilities/EventsFileManager.kt

```diff
@@ -120,6 +124,8 @@ class EventsFileManager(
         val contents = """],"sentAt":"${Instant.now()}"}"""
```

Comment:
@prayansh wonder if we should replace `Instant.now()` with `SimpleDateFormat` to be back compatible. this call is only available on android api 26 or above

Reply:
we dont need to do that because we are using the `coreLibraryDesguaring` apis, which allows us to use Instant in older API levels.